live-ingester/src/integration-test/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/e2e/repository/MatchingContentMappingRequestIntegrationTest.java

```diff
@@ -124,8 +124,8 @@ void givenMappingForImage_whenContentWithMatchingTypeIngested_thenProcessWithTra
                           }
                         }
                       },
-                      "ALLOW_ACCESS": {"value": ["GROUP_EVERYONE"]},
-                      "DENY_ACCESS": {"value": []}
+                      "ALLOW_ACCESS": {"type": "string", "value": ["GROUP_EVERYONE"]},
+                      "DENY_ACCESS": {"type": "string", "value": []}
```

Comment:
This raises an interesting question - how can we guess the type for an empty array. Anyway, for now I suggest we assume `string` unless we can immediately tell otherwise from the value.